ui/components/ErrorPage.js

```diff
@@ -1,96 +1,93 @@
 import React, { useState, useEffect } from 'react';
-import { NoSsr, Typography, makeStyles, Link } from '@material-ui/core';
+import { NoSsr } from '@material-ui/core';
+import { Typography, Link, Box, styled, useTheme } from '@layer5/sistent';
+import { UsesSistent } from './SistentWrapper';
 
-const styles = makeStyles((theme) => ({
-  rootClass: {
-    backgroundColor: theme.palette.type === 'dark' ? theme.palette.secondary.headerColor : '#fff',
-    padding: '2rem',
-    textAlign: 'center',
-    borderRadius: 4,
-    height: '100%',
-  },
-  errorSection: {},
-  message: {
-    fontSize: '3rem',
-    lineHeight: '2rem',
-    marginBottom: '2rem',
-  },
-  errMessage: {
-    fontWeight: '400',
-    fontSize: '1.5rem',
-    color: 'gray',
-    fontStyle: 'italic',
-    marginTop: '2.5rem',
-  },
-
-  discussionforumlink: {
-    color: theme.palette.type === 'dark' ? '#00B39F' : '#',
-  },
-
-  mesh: {
-    display: 'block',
-    margin: 'auto',
-    marginTop: '3.125rem',
-    maxWidth: '50%',
-    height: '45%',
-  },
-  helpMessage: {
-    marginTop: '5rem',
-    color: theme.palette.type === 'dark' ? '#fff' : '#222',
-  },
+const StyledBox = styled(Box)(({ theme }) => ({
+  backgroundColor: theme.palette.mode === 'dark' ? theme.palette.background?.default : '#fff',
```

Comment:
this is not right way of using theme, if you are using background default it already take cares of dark and light mode.
```suggestion
  backgroundColor: theme.palette.background?.default,
```